reactive/kotlinx-coroutines-reactive/src/module-info.java

```diff
@@ -0,0 +1,8 @@
+module kotlinx.coroutines.reactive {
+    requires kotlin.stdlib;
+    requires kotlinx.coroutines.core;
+    requires kotlinx.atomicfu;
```

Comment:
All `atomicfu` mentions should be erased at the runtime, so this one should not be necessary

Reply:
There are multiple codepaths where `kotlinx.atomicfu.AtomicRef` is used, which causes the requires on `atomicfu` to be needed. Without it, the module system can't check / guarantee that `atomicfu` is on the classpath (or more correctly, the module path) which could cause a `ClassNotFoundException` during runtime.